src/pages/index.js

```diff
@@ -65,34 +89,166 @@ cardAddButton.addEventListener("click", () => {
   cardAddModal.open();
 });
 
-/* Profile Edit & Card Add Functions */
+/* Profile Edit */
+
+const editFormValidator = new FormValidator(
+  validationSettings,
+  profileEditForm
+);
+
+const profileEditModal = new PopupWithForm(
+  "#profile-edit-modal",
+  handleProfileEditSubmit
+);
 
 function handleProfileEditSubmit(data) {
-  userInfo.setUserInfo(data);
-  profileEditModal.close();
+  profileEditModal.renderLoading(true);
+  api
+    .updateProfileInfo({ name: data.title, about: data.description })
+    .then(() => {
+      userInfo.setUserInfo(data);
+      profileEditModal.close();
+      editFormValidator.disableSubmitButton();
+    })
+    .catch((err) => {
+      console.error("Could not update profile info:", err);
+    })
+    .finally(() => {
+      profileEditModal.renderLoading(false);
+    });
   editFormValidator.disableSubmitButton();
```

Comment:
`  editFormValidator.disableSubmitButton();
`

Please, delete it

You do it in line 111